packages/@o3r/new-version/src/index.ts

```diff
@@ -136,7 +137,7 @@ export class NewVersion {
     // Sort tags in descending order and exclude next major in preparation
     let parsedSortedTags = tags
       .map((tag) => semver.parse(tag.replace('V', 'v')))
-      .filter((tag): tag is semver.SemVer => !!tag && !!tag.raw.match(`^(?:v)?${versionMask}`))
+      .filter((tag): tag is semver.SemVer => !!tag && !!new RegExp(`^(?:v)?${versionMask}`).test(tag.raw))
```

Comment:
why not be also why? :D

the regex should me moved out to the `filter` loop in a const

Reply:
[unicorn](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-regexp-test.md)

Reply:
regexp moved outside the filter loop in a const